### DISCUSSION TITLE
Proposal: please delete.

### DISCUSSION THREAD

**Discussion by Berny-art** [akash-network/community#755] [Governance Proposals]
## Overview

Proposal is not needed anymore.

**Comment by nick134-bit** [comment]
GM, if there is demand for this pool there will be liquidity. This should be no problem, like we saw recently with SPICE/AKT Pools on Osmosis. I don't think a community pool spend is needed here.

**Comment by rodri-r** [comment]
Nice proposal,idea and project.  

While I like the idea of having NFTs being traded with AKT, I agree with what @nick134-bit mentions above about liquidity not being a problem once these NFTs become purchasable.  

I also think that the $12.5K in STARS that you are suggesting for the LP should not come from the Akash community pool, but instead these should be provided by the Stargaze team/community or someone else, as IMO adding AKT to their market to trade NFTs will also benefit their network and this should be worked on by both communities.  

What you suggest is to get $25k from Akash community pool, selling half of that to buy Stars and thus this will benefit their token and affect the AKT token, which I am not in favor of.

**Comment by piber-dev** [comment]
hello @Berny-art thanks for writing this proposal. could you elaborate a little more on the marketing planned here? i'd love to be able to understand more of the tangible results this can provide for akash besides the main art part of this. i believe that the marketing details is what is probably the most interesting here as that can give a real return on investment more than just using funds on internet culture.
